Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM node:12.16.3
+LABEL maintainer=Netlify
+
+RUN apt update && apt install -y jq
+
+COPY plugins.json .
+RUN jq -r 'map(.package + "@" + .version) | join(" ")' plugins.json | xargs npm install
```

Comment:
Love the usage of `jq` + `xargs` to do this in a single line.

Reply:
Thank you, I was really happy with it! Resolves https://github.com/netlify/build-image-plugins/issues/18 as well.